docs/index.md

```diff
@@ -23,4 +23,5 @@ provider "scaffolding" {
 
 ### Optional
 
-- `example` (String) Example provider attribute
+- `base_api_url` (String) Used by api client to connect to flagsmith instance. NOTE: update this if you are running a self hosted version
+- `master_api_key` (String, Sensitive) Master API key used by flagsmith api client. Can also be set using the environment variable `FLAGSMITH_MASTER_API_KEY`
```

Comment:
That's actually  a common pattern to support passing `FLAGSMITH_MASTER_API_KEY` from the environment variable, because it's sensitive

Reply:
Well, because it is optional, else there is no way for us to read it from environment. If it's required, terraform will make sure it's in `.tf` hence optional